org.azahar_emu.Azahar.json

```diff
@@ -0,0 +1,140 @@
+{
+    "app-id": "org.azahar_emu.Azahar",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.llvm18"
+    ],
+    "command": "azahar-launcher",
+    "rename-desktop-file": "azahar.desktop",
```

Comment:
In azahar-emu/azah#582 you wrote

> To my knowledge this is not an upstream requirement

but now you are renaming the files. I highly recommend you to change the name upstream instead of here. Some things in Linux (e.g. the window icon on some Wayland compositors) are tied to the desktop file, so this would avoid problems.

Reply:
Alright then, I'll update the desktop file name for the next minor update

Reply:
Would be nice if you update the other names too

Reply:
> Would be nice if you update the other names too

Other names? Like the launcher script?